src/actions/profile/types.ts

```diff
@@ -16,3 +16,5 @@ export const GET_USDC_BALANCE = 'GET_USDC_BALANCE';
 export const GET_PROFILE_CONTRACT_ADDRESS = 'GET_PROFILE_CONTRACT_ADDRESS';
 export const UPDATE_PROFILE_CONTRACT_ADDRESS =
   'UPDATE_PROFILE_CONTRACT_ADDRESS';
+export const GET_CURRENT_USER_PROFILE_PICTURE =
+  'UPDATE_PROFILE_CONTRACT_ADDRESS';
```

Comment:
The string value `'UPDATE_PROFILE_CONTRACT_ADDRESS'` is wrong, again should be set